.changeset/khaki-hounds-think.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-actions': minor
```

Comment:
```suggestion
'@backstage/plugin-github-actions': patch
```

Since this isn't a breaking change we can keep this a patch since the plugin is still on 0.x

Reply:
Updated